docker-compose.yml

```diff
@@ -1,18 +1,24 @@
 version: '3'
 services:
   redis:
+    container_name: redis
```

Comment:
Why do you override this? The default (a docker compose generated one) is usually preferred because if you have multiple compose stacks running simultaneously, you can distinguish which stack a container belongs to.
There might be a good reason to do so though, so curious why you opt for this.